src/main/java/uk/gov/hmcts/reform/coh/controller/question/QuestionResponse.java

```diff
@@ -19,6 +19,9 @@
     @JsonInclude(JsonInclude.Include.NON_NULL)
     private String deadlineExpiryDate;
 
+    @JsonProperty(value = "deadline_extension_count")
+    private String deadlineExtCount;
```

Comment:
Why is this stored as a String?

Reply:
It's because the mapper only accepts a string. I suppose I could move it out of the QuestionResponseMapper.QUESTION_DEADLINE_EXT_COUNT enum?

Reply:
It's inconsistent with the the question round counter which is an Integer. I think you should have them both the same either way.